src/Bundle/DependencyInjection/Configuration.php

```diff
@@ -39,6 +39,9 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->arrayNode('mapping')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->arrayNode('imports')
```

Comment:
Do you have example how it will be used? We now have both `imports` and `paths`, where `imports` are paths for php only configs?

Reply:
Yes look at the APIP PR for the whole explanation. Basically we need files that are not in the autoloader.